backend/core/views.py

```diff
@@ -9,7 +9,9 @@
 @csrf_exempt
 def stripe_webhook(request) :
     payload = request.body
-    sig_header = request.META['HTTP_STRIPE_SIGNATURE']
+    sig_header = request.META.get('HTTP_STRIPE_SIGNATURE', None)
```

Comment:
@arminpatel Why do we need this change?

Reply:
So we can return a 400 error, otherwise it would return a 500